ml-agents/mlagents/trainers/learn.py

```diff
@@ -450,8 +451,8 @@ def run_cli(options: RunOptions) -> None:
     except Exception:
         print("\n\n\tUnity Technologies\n")
     print(get_version_string())
-    trainer_logger = logging.getLogger("mlagents.trainers")
-    env_logger = logging.getLogger("mlagents_envs")
+    trainer_logger = create_logger("mlagents.trainers")
+    env_logger = create_logger("mlagents_envs")
 
     if options.debug:
         trainer_logger.setLevel("DEBUG")
```

Comment:
We might be able to delete the `else` in this block since we're setting the level in `create_logger`

Reply:
@ervteng are you suggesting that we make `tf_utils.set_warnings_enabled(False)` the default?